database/tool.gpt

```diff
@@ -3,32 +3,33 @@ Name: Database
 Description: Tools for interacting with a database
 Metadata: category: Capability
 Metadata: icon: https://cdn.jsdelivr.net/npm/@phosphor-icons/core@2/assets/duotone/database-duotone.svg
-Share Tools: Run Database Query, Exec Database Statement
+Share Tools: Run Database Command
 
 ---
-Name: List Database Tables
-Description: List all tables in the SQLite database and return a JSON object containing the results
+Name: Run Database Command
+Share Context: Database Context
+Description: Run the sqlite3 command with the given arguments against the SQLite database and print the results
+Param: sqlite3_args: Arguments to pass to the sqlite3 cli
 
-#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool listDatabaseTables
+#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool runDatabaseCommand
 
 ---
-Name: Run Database Query
-Description: Run a SQL query against the SQLite database and return a JSON object containing the results
-Share Context: Database Context
-Param: query: SQL query to run
+Name: Database Context
+Type: context
 
-#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool runDatabaseQuery
+#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool databaseContext
 
 ---
-Name: Exec Database Statement
-Description: Execute a SQL statement against the SQLite database
-Share Context: Database Context
-Param: statement: SQL statement to execute
+Name: list_database_tables
+Metadata: index: false
+Description: List all tables in the SQLite database and return a JSON object containing the results
 
-#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool execDatabaseStatement
+#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool listDatabaseTables
 
 ---
-Name: Database Context
-Type: context
+Name: list_database_table_rows
```

Comment:
Are these two tools used anywhere?

Reply:
yes they're used by the UI. There's another PR to switch over to this.